metal/mmtl/launch.py

```diff
@@ -71,6 +80,12 @@ def get_dir_name(models_dir):
             "This is used, e.g., for tuning."
         ),
     )
+
+    # Use auxiliarty tasks
+    parser.add_argument(
+        "--use_auxiliary", type=int, default=0, help="Use auxiliary tasks or not"
```

Comment:
I'd make the default Bool, just to be clear. It'll still accept 0 and 1 as values.

Reply:
Done